clang/test/CodeGen/ubsan-shift-bitint.c

```diff
@@ -0,0 +1,36 @@
+// RUN: %clang_cc1 %s -O0 -fsanitize=shift-exponent -emit-llvm -o - | FileCheck %s
```

Comment:
I think you for example want to use `-std=c2x -triple=x86_64-unknown-linux` here.
(Not sure exactly if you for example need the triple, but I fear that you would end up with failing build bots otherwise as not all targets support C23 _BitInt literals by default.)

Reply:
Sounds good, thank you for the recommendation!